fe/fe-core/src/test/java/org/apache/doris/nereids/AnalyzeSSBTest.java

```diff
@@ -161,10 +161,7 @@ private void executeRewriteBottomUpJob(PlannerContext plannerContext, RuleFactor
     }
 
     private boolean checkBound(LogicalPlan root) {
-        if (!checkPlanBound(root))  {
-            return false;
-        }
-        return true;
+        return checkPlanBound(root);
```

Comment:
I think we can remove checkBound or checkPlanBound  function

Reply:
i think so